### DISCUSSION TITLE
Swiss FSP Audio Streaming

### DISCUSSION THREAD

**Discussion by fc-64** [emukidid/swiss-gc#672] [Q&A] [answered]
I'm running .6r1318, and connected to an FSP server running on Docker.  So far, all games without audio streaming load without issue.

However, every game with audio streaming presents the following error when loading:

Exhausted Reserved Memory
An SD Card Adapter is necessary in order for patches to reserve additional memory.

I am currently using an SD adapter in memory card slot B, which is where the Swiss settings are stored.  The only settings I've changed from the default is the Swiss video mode is set to 480i, stop the DVD motor, and the FSP server.  The memory card is only 2GB, but has more than enough free space to house an entire disc image, let alone the patches that are applied when starting a game.

I am booting Swiss via the Wind Waker exploit.

Are there any other recommendations or changes I've neglected to make?

**Comment by Extrems** [comment, marked as answer]
See issue #665.

**Reply by fc-64** [reply]
Classic FSP!  Sorry my Google-fu wasn't up to snuff this time.  Thanks for your continued support and dedication to this project!
